src/libs/typesystem/SpecTranslator/Elektra/Parsers.hs

```diff
@@ -0,0 +1,122 @@
+--
+-- @file
+--
+-- @brief Some parsers which are used in the translation process
+--
+-- @copyright BSD License (see LICENSE.md or https://www.libelektra.org)
+-- 
+{-# LANGUAGE TypeFamilies, TypeInType, TypeOperators, OverloadedStrings,
+             ExistentialQuantification, GADTs, UndecidableInstances #-}
+
+module Elektra.Parsers (parseTypeSignature, parseRange, 
+  regexTypeP, regexTypeParameterP, typeSignatureP, rangeP) where
+
+import Control.Applicative (empty)
+import Data.Void
+
+import Elektra.Specifications
+
+import Text.Megaparsec
+import Text.Megaparsec.Char
+import qualified Text.Megaparsec.Char.Lexer as L
+
+type Parser = Parsec Void String
+
+-- Accessory functions
```

Comment:
from?

Reply:
... how do you mean from?

Reply:
The comment was not useful for me.

Accessory functions for <?>